.dockerignore

```diff
@@ -1,5 +1,4 @@
 .DS_Store
-.git
```

Comment:
Curious; do you know if search engines actually complained about this? As the last-modified date is _optional_ https://www.sitemaps.org/protocol.html

So the alternative could be to have a global `changefreq` included at which search engines should re-index 🤔

Reply:
> Does it require git for this?

Yes to have proper metadata as `mtime` will not be accurate when repo is cloned on GitHub Runners.

> This may have quite an impact on the build-cache when building locally 😬

I don't see a huge impact on my side or on GitHub atm. We jump from ~36s to ~42s.

If you check the `RUN` cmd mounting the workspace it's quite fast: https://github.com/docker/docker.github.io/runs/7581677164?check_suite_focus=true#step:4:342

Let me know if you see some perf regression on your side.

Reply:
I can try later, but thinking of scenarios where the `.git` metadata is changed, but no changes in files (e.g. after I would create a new branch and switch back to the branch, it would also invalidate the current branch, even if there's no changes in any of the documents?)

Reply:
In addition to the extra size 😬 

```bash
du -sh ./.git
721M	./.git
```

Reply:
The `generate` stage is not cached at all atm to cover changes from remote resources so doesn't change the current behavior: https://github.com/docker/docker.github.io/blob/90b8f0249e942fb3010b7bce44820a4e3703d9a3/docker-bake.hcl#L16

On my side almost the same:

```console
$ du -sh ./.git
674M    ./.git
```